docs/source/tutorials/plug-and-charge.rst

```diff
@@ -0,0 +1,128 @@
+.. _tutorial_plug_and_charge:
+
+#####################################################
+Plug&Charge with EVerest Software in the loop
+#####################################################
+
+EVerest provides support for Plug&Charge within ISO15118-2 and OCPP1.6 and
+OCPP2.0.1. This tutorial explains how you can set up and configure EVerest
+for Plug&Charge with the software in the loop.
+
+.. _prerequisites:
+
+Prerequisites
+=============
+
+If you're new to EVerest, start with our
+`Quick Start Guide <02_quick_start_guide.html>`_ to get a simulation in
+EVerest running for the first time.
+If you have done that successfully, you can move on with this tutorial.
+
+If you are interested in more in-depth information about how Plug&Charge is
+implemented in EVerest, please refer to the 
+:doc:`Plug&Charge explanations <explanation/pnc-process>`
+and the :doc:`Plug&Charge Configuration Howto <how-to-guides/configure-pnc>`
+
+Packages for ISO 15118 communication
+====================================
+
+To be able to build EVerest with ISO 15118 capability, you will have to
+install the requirements for Josev:
+
+.. code-block:: bash
+  cd {EVerest Workspace Directory}/Josev
+  python3 -m pip install -r requirements.txt
+
+For ISO 15118 communication including Plug&Charge, install Josev and some CA
+certificates:
+
+.. code-block:: bash
+  cd {EVerest Workspace Directory}/Josev/iso15118/shared/pki
+  ./create_certs.sh -v iso-2 -i {EVerest Workspace Directory}/everest-core
```

Comment:
Code blocks do not render:
<img width="1662" height="498" alt="image" src="https://github.com/user-attachments/assets/93e7ce0b-e3b7-48b1-9e74-5742df53a09e" />